src/AdminCli/Infrastructure/Logging.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using AdminCli.Configuration;
+using Serilog;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace AdminCli.Infrastructure;
+
+public static class Logging
+{
+    private static LoggingLevelSwitch LoggingLevelSwitch { get; } = new (LogEventLevel.Warning);
+
+    public static ILogger CreateLogger()
+    {
```

Comment:
I choose Serilog because it is able to change the log level while the app executes. [Microsoft.Extensions.Logging does not support this scenario](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#change-log-levels-in-a-running-app). Especially if we want `--log-level` to be optional for every command, we cannot use Microsoft.Extensions.Loggin.

Reply:
We don't have to change the level at runtime. Because the app gets bootstrapped with the necessary level so we can incorporate it while building the logger

Reply:
rough **scratch** not taking the CLI library into context, just plain old .NET. No error handling or anything else 😄 

```
var lvl = args[0];
using var loggerFactory = LoggerFactory.Create(builder =>
{
  builder
  .SetMinimumLevel(Enum.Parse<LogLevel>(lvl, true))
  .AddConsole();
});
var l = loggerFactory.CreateLogger<Program>();
```

Reply:
Yeah, if we bypass McMaster.Extensions.CommandLineUtils, then it's possible. I will remove Serilog.